### PR TITLE
Added GA events for tracking a few clicks

### PR DESCRIPTION
**Pull request by jonasfj**

### CODE REVIEW

**Comment by jonasfj** [comment]
Should fix #3253.

File: static/js/gtag.js [hunk 1]
```diff
@@ -1,4 +1,39 @@
...
+        return null;
+      }
+      return 'package:' + pkg;
+    }
+    return null;
+  }
+  var label = findPackageName() || '-';
+
+  function sendEvent(e) {
+    gtag('event', e.currentTarget.dataset.gaClickEvent, {
```
Comment (isoos) [inline static/js/gtag.js]: Please add a comment here that currentTarget is the DOM Element where the event listener was added to, and not the Element that the click happened.

File: static/js/gtag.js [hunk 2]
```diff
@@ -1,4 +1,39 @@
...
+      'event_category': 'click',
+      'event_label': label,
```
Comment (isoos) [inline static/js/gtag.js]: Can we have a 'package' parameter instead? Or let's add `package:` prefix to the value, so publishers can also be tracked similarly.

Comment (jonasfj) [reply static/js/gtag.js]: We already have a package prefix added in `findPackageName`.

As I understood events: category is for sorting... action is the thing we count, and label (optional) is a bit of extra context..

On a given session an action can ONLY be counted once.. so label shouldn't necessarily be thought of as an extra level...

Okay, maybe I only somewhat understand the docs... :)

Comment (jonasfj) [reply static/js/gtag.js]: Maybe I should move the prefixing, so that it's less obscure... Or what are you suggesting..

Comment (isoos) [reply static/js/gtag.js]: Maybe we should emit the value that we want to have here in the main HTML, so no lookup is needed?

Comment (jonasfj) [reply static/js/gtag.js]: My thinking was to use the page `<title>` at one point... or maybe the path from the URL, but stripping away query string and hashes...

**Pull request merged**